### PR TITLE
Add Uploader 2.6.2 to release-notes.md

### PR DESCRIPTION
**Pull request by christopherasnider**
- Fixed typos and formatting (still)
- Added Uploader 2.6.2 notes.

### CODE REVIEW

File: _docs/release-notes.md [hunk 1]
```diff
@@ -306,7 +306,19 @@ Visit [https://tidepool.org/uploader](https://tidepool.org/uploader) to download
 
 If you already have the Tidepool Uploader installed on your computer, it will automatically update to the latest version.  
 
-### 2.6.0 (Release 2018-06-04)
+### 2.6.2 (Released 2018-06-13)
+We fixed some of our code that deals with identifying the right time for your data. If you really want to dig into our UTC Bootstrapping algorithm, send us an email and we'll talk your ear off about the challenges of device times.  
+[Feature Requirements](https://trello.com/c/rh2oLpjR/8-apply-utc-bootstrapping-fix-for-invalid-timezone-offset-to-new-data)
+
+We're running on node-serialport v6 now. You may not think that's exciting, but now devices that connect through a serial port will automatically retry an upload if it fails the first time. And you thought dependency updates were boring.
```
Comment (gniezen) [inline _docs/release-notes.md]: I realise this is a bit pedantic, but the auto-retry code is something I added after updating to v6.

Comment (christopherasnider) [reply _docs/release-notes.md]: How about: "You may not think that's exciting, but this update let us add code to automatically retry an upload..."

Comment (christopherasnider) [reply _docs/release-notes.md]: Full edit: "We're running on node-serialport v6 now. You may not think that's exciting, but this update let us add code to automatically retry an upload the first time a device connected through a serial port fails. And you thought dependency updates were boring."

Comment (gniezen) [reply _docs/release-notes.md]: Maybe "You may not think that's exciting, but it fixes a bunch of issues for devices that connect through a serial port on Windows. And you thought dependency updates were boring. We now also automatically retry a serial port upload if it fails the first time. "?

Comment (christopherasnider) [reply _docs/release-notes.md]: I like that. I'll make the update and merge the PR.

**Pull request merged**